### PR TITLE
Fix if mouse leaves Tooltip.Content and moves over an iframe, the Radix Tooltip stays open

### PR DESCRIPTION
**Pull request by istarkov**
## Description

When the mouse leaves Tooltip.Content and moves over an iframe, the Radix Tooltip stays open.
This happens because Radix's internal grace area relies on the pointermove event, which isn't triggered over iframes.
The current workaround is to set pointer-events: none on the canvas when the tooltip is open.

Here are:
- [x] Tooltip component sets `--canvas-pointer-events` on the documentElement if any tooltip is open
- [x] Iframe uses `--canvas-pointer-events`
- [x] `isCanvasPointerEventsEnabledStore` in order to use above css variable
- [x] Color picker hack fully removed. Now just intercepts `open` state change, as it's dialog and  canvas is not interactive when it's open already so it's safe to disable pointer-events too.

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/builder/features/settings-panel/settings-panel.tsx [hunk 1]
```diff
@@ -13,6 +14,10 @@ export const SettingsPanelContainer = ({
   return (
     <>
       <SettingsSection />
+      <Tooltip defaultOpen={true} content="This is a tooltip">
```
Comment (kof) [inline apps/builder/app/builder/features/settings-panel/settings-panel.tsx:17]: is this forgotten?

File: apps/builder/app/builder/features/style-panel/shared/color-picker.tsx [hunk 2]
```diff
@@ -88,35 +88,6 @@ const styleValueToRgbaColor = (value: CssColorPickerValueInput): RgbaColor => {
   };
 };
 
-// Dragging over canvas iframe with CORS policies will lead to loosing events and getting stuck in mousedown state.
```
Comment (kof) [inline apps/builder/app/builder/features/style-panel/shared/color-picker.tsx:91]: why is this removed? the problem is still there

**Comment by kof** [comment]
- [ ] not created in this issue but the tooltip around color picker is broken

Screenshot 2023-08-09 at 19 43 07

File: apps/builder/app/builder/features/style-panel/shared/color-picker.tsx (hunk 2 above)
Comment (kof) [reply apps/builder/app/builder/features/style-panel/shared/color-picker.tsx:91]: just tested, seems to work

**Review by kof — APPROVED** [review]
don't forget to remove the TEST thing
